ui-v2/app/adapters/acl.js

```diff
@@ -19,6 +19,9 @@ export default Adapter.extend({
   },
   urlForQueryRecord: function(query, modelName) {
     // https://www.consul.io/api/acl.html#read-acl-token
+    if (typeof query.id === 'undefined') {
+      throw new Error('You must specify an id');
+    }
     return this.appendURL('acl/info', [query.id], this.cleanQuery(query));
```

Comment:
Is `appendURL` still needed too? I thought things get url encoded higher up.

Reply:
Yeah it was just an attempt to make things more ember-y. It just calls an pure imported function from the ApplicationAdapter. The function encodes the query strings, paths are encode higher still as far as I remember. I'm considering just hoisting that import up so its clearer, although I'd have to import it everywhere, so not sure yet.